cccs-build/superset/Dockerfile

```
@@ -1,8 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230605182047_b6972
-
+FROM uchimera.azurecr.io/cccs/superset-base:feature_dataset-explorer-chart_20230621174917_b7116
```

```suggestion
FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.1_20230605182047_b6972
```
This shouldn't be changed in this PR. However, it will need to be updated in another PR once this one is merged in and a new `cccs/superset-base` tag is created.

---

This has since been resolved.